tensorflow/core/platform/default/logging.h

```diff
@@ -71,8 +72,8 @@ class LogMessageFatal : public LogMessage {
 
 #define LOG(severity) _TF_LOG_##severity
 
-// TODO(jeff): Define a proper implementation of VLOG_IS_ON
-#define VLOG_IS_ON(lvl) ((lvl) <= 0)
+// Set TF_CPP_MIN_VLOG_LEVEL environment to update minimum log level of VLOG
+#define VLOG_IS_ON(lvl) ((lvl) <= ::tensorflow::internal::LogMessage::min_lvl_)
 
 #define VLOG(lvl)      \
   if (VLOG_IS_ON(lvl)) \
```

Comment:
similar to the normal log level, can you add

if (TF_PREDICT_FALSE(VLOG_IS_ON(lvl))) ?

(Most of the time, vlogging will be off, so it's nice to provide the compiler hint).